docs/development.md

```diff
@@ -231,16 +296,24 @@ to the main katello server you want the client attached to
 
 ## Webpack
 
-Forklift creates a reverse proxy between localhost:3000 (Rails development default server) and https://forkliftfqdn (e.g: https://centos7-devel.example.com). This will cause mixed-content errors if you try to enable the webpack server without any options, as it needs to use the right certificates for HTTPS and Foreman needs to allow it. If you do not need to hack on Webpack code, ensure `:webpack_dev_server` is set to false in `~/foreman/config/settings.yaml` and run `bundle exec rake webpack:compile` from `~/foreman`.That will create the necessary assets to be served by Foreman. If you want to work on Webpack code, follow these instructions:
+Forklift creates a reverse proxy between localhost:3000 (Rails development default server) and https://forkliftfqdn (e.g: https://centos7-devel.example.com). This will cause mixed-content errors if you try to enable the webpack server without any options, as it needs to use the right certificates for HTTPS and Foreman needs to allow it. Thus, to use the reverse proxy you need to follow these additional steps (while connected to the guest's shell):
 
-To work with webpack-dev-server, ensure the following lines are in your `~/foreman/config/settings.yaml`:
+Add/change the following variables in  `~/foreman/config/settings.yaml`:
 
 ```yaml
 :webpack_dev_server: true
 :webpack_dev_server_https: true
 ```
 
-This will instruct Foreman to use Webpack from an https source. Now we need to make sure Webpack is using the right certificates to serve the development content through HTTPS. We're going to save these configuration values on `~/foreman/.env`:
+This will instruct Foreman to use Webpack from an https source.
+
+Now recompile webpack:
+
+```
+bundle exec rake webpack:compile
```

Comment:
I do not think you need to compile again. When using `foreman start`, webpack compiles the assets and servers them on the fly.